packages/form-footer/src/FormFooter.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useMemo } from 'react';
+import Banner from '@leafygreen-ui/banner';
+import Button from '@leafygreen-ui/button';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { isComponentType } from '@leafygreen-ui/lib';
+import { once } from 'lodash';
+
+const footerStyle = (sticky: boolean) => {
+  return css`
+    position: ${sticky ? 'fixed' : 'static'};
+    bottom: ${sticky ? '0px' : 'unset'};
+    left: ${sticky ? '0px' : 'unset'};
+    min-height: 92px;
+    width: 100%;
+    padding: 24px;
+    border: 1px solid ${uiColors.gray.light2};
+    box-shadow: 0px -4px 4px 0px rgba(6, 22, 33, 0.1);
+  `;
+};
+
+const contentStyle = css`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  gap: 8px;
+
+  > button {
+    white-space: nowrap;
+  }
+`;
+
+const flexSpan = css`
+  display: inline-flex;
+  gap: inherit;
+`;
+
+const bannerStyle = css`
+  flex-grow: 0;
+  min-height: unset;
+  padding-block: 7px;
+`;
+
+/**
+ * Types
+ */
+interface PrimaryButtonProps {
+  text: string;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+  variant?: 'primary' | 'danger';
+  disabled?: boolean;
+  type?: 'button' | 'submit';
+}
+
+const isPrimaryButtonProps = (testObj: any): testObj is PrimaryButtonProps => {
+  return testObj && testObj.text != null;
+};
+
+export interface FormFooterProps {
+  /**
+   * Defines whether the footer should be "stuck" to the bottom of the frame.
+   * Can also be customized using `className`
+   */
+  sticky?: boolean;
```

Comment:
I worry this will be confused with `position: sticky`. Do we need to handle this ourselves in the first place? It seems simple enough for folks to apply the positioning they need for their own case.

Reply:
That's fair—I was having similar thoughts. I'll remove this prop